### PR TITLE
Add sticker_items and deprecate stickers

### PR DESCRIPTION
**Pull request by Yomanz**
Documentation PR: https://github.com/discord/discord-api-docs/pull/3171

### CODE REVIEW

**Comment by Yomanz** [comment]
> 
> 
> Sigh. Tempted to just say remove the old stuff completely

I can remove it entirely if you want, it's only present on past messages iirc

File: data/src/main/scala/ackcord/data/DiscordProtocol.scala [hunk 1]
```diff
@@ -25,11 +25,9 @@ package ackcord.data
 
 import ackcord.data.AuditLogChange.PartialRole
 import ackcord.data.raw._
-import cats.syntax.all._
```
Comment (Katrix) [inline data/src/main/scala/ackcord/data/DiscordProtocol.scala:28]: Removing this import causes 2.12 to fail to compile.

**Comment by Katrix** [comment]
> 
> 
> > Sigh. Tempted to just say remove the old stuff completely
> 
> I can remove it entirely if you want, it's only present on past messages iirc

Hmmm. If it's still present in old messages I'm even more conflicted. Just add a since tag on the deprecation message for now, and I'll remove it later I'd say. My general stance is to stop supporting stuff a bit after Discord stops playing nice with it.

File: gatewayData/src/main/scala/ackcord/gateway/GatewayProtocol.scala [hunk 2]
```diff
@@ -25,9 +25,8 @@ package ackcord.gateway
 
 import ackcord.data._
 import ackcord.data.raw.{PartialRawGuildMember, RawMessageActivity}
-import ackcord.util.{JsonNull, JsonOption, JsonSome, JsonUndefined}
+import ackcord.util._
 import cats.Later
-import cats.syntax.all._
```
Comment (Katrix) [inline gatewayData/src/main/scala/ackcord/gateway/GatewayProtocol.scala:30]: Need to not remove this import as well.

**Pull request merged**